utils/getCrimeData.js

```diff
@@ -0,0 +1,70 @@
+export class CrimeData {
+    constructor(latitude, longitude, monthFrom = "", monthTo = "") {
+        this.latitude = latitude;
+        this.longitude = longitude;
+        this.monthFrom = monthFrom;
+        this.monthTo = monthTo;
+        this.crimeIncidents = [];
+    }
+
+    get total() {
+        return this.crimeIncidents.length;
+    }
+
+    async fetchCrimeData() {
+
+        const baseUrl = "https://data.police.uk/api/crimes-street/all-crime";
+
+        let urlWithQueryString = `${baseUrl}?lat=${this.latitude}&lng=${this.longitude}`;
+        // Add date to query string if specified in function call
+        urlWithQueryString += (this.monthFrom == "") ? "" : `&date=${this.monthFrom}`;
+
+        try {
+            const response = await fetch(urlWithQueryString);
+            
+            if (response.ok) {
+                const crimeDataJson = await response.json();
+                this.createCrimeDataFromJson(crimeDataJson);
+                return this;
+            }
+            else {
+                throw new Error(`Unable to fetch crime data:\n${response.status}`);
+            }
+        }
+        catch (error) {
+            throw error;
+        }
+    }
+
+    createCrimeDataFromJson(crimeDataJson) {
+        this.crimeIncidents = [];
```

Comment:
you have already initialized this variable at the top. I commented it out and the program worked fine.